sunpy/coordinates/transformations.py

```diff
@@ -197,52 +213,64 @@ def hpc_to_hpc(heliopcoord, heliopframe):
     return hpc
 
 
-# Make a transformation graph for the documentation, borrowed lovingly from
-# Astropy.
+def _make_rotation_matrix(start_representation, end_representation):
```

Comment:
I'd suggest a slightly more specific name like `_make_rotation_matrix_from_reprs`.  When reading over other code I found just `_make_rotation_matrix` confusing because it's very close to an equally common case of a rotation matrix from an axis and an angle.